schedule/rt/sle15/rt-product-textmode-autoyast.yaml

```diff
@@ -0,0 +1,18 @@
+name: autoyast_rt_textmode
```

Comment:
i would excepted to find `rt-product-textmode-autoyast` as name convention from  `rt-product-gnome-autoyast`. would you mind to change it?